doc/source/cluster/key-concepts.rst

```diff
@@ -60,19 +60,19 @@ sit idle, the autoscaler will remove worker nodes from the cluster.
 It is important to understand that the autoscaler only reacts to task and actor resource requests, and not application metrics or physical resource utilization.
 To learn more about autoscaling, refer to the user guides for Ray clusters on :ref:`VMs <cloud-vm-index>` and :ref:`Kubernetes <kuberay-index>`.
 
+
 .. _cluster-clients-and-jobs:
 
-Ray Jobs
---------
+Ray Job
+-------
 
-The main method for running a workload on a Ray cluster is to use Ray Jobs.
-Ray Jobs enable users to submit locally developed-and-tested applications to a
-remote Ray cluster. Ray Job Submission simplifies the experience of packaging,
-deploying, and managing a Ray application.
+A Ray job is a single application: it is the collection of Ray tasks, objects, and actors that originate from the same script.
+The worker that runs the Python script is known as the *driver* of the job.
 
-For interactive development, the following additional methods are available:
+There are three ways to run a Ray job on a Ray cluster:
 
-* Directly running a script or notebook on any head or worker node.
-* Using the Ray Client to connect remotely to the cluster.
+1. (Recommended) Submit the job using the :ref:`Ray Jobs API <jobs-overview>`.
+2. Run the driver script directly on any node of the Ray cluster.
+3. Use :ref:`Ray Client <ray-client-ref>` to connect remotely to the cluster within a driver script, for interactive development.
```

Comment:
I think we should move `, for interactive development.` to bullet point 2 since we want to de-emphasize ray client.